### PR TITLE
BUGFIX: Make the login screen more adaptive to changes

### PR DESCRIPTION
**Pull request by gerhard-boden**
Complete overhaul for the login screen element alignment by making use of `flexible boxes`. Especially smaller devices will benefit from this and everyone involved in theming the login screen. This also fixes the login screen mobile view.

### CODE REVIEW

**Comment by gerhard-boden** [comment]
I know I'm kinda dating myself here, but this change may seem subtle on a desktop resolution, but will make things a lot easier when making future adjustments or when it comes to theming (like https://github.com/neos/neos-development-collection/pull/545). Since there are no more `position: absolute` footers or unnecessary elements with fixed height or width left, inserting or changing stuff is much more fun now.

Furthermore the current version is broken on smaller devices:
image
image

Now, if someone could go ahead and make the small rest of the backend mobile friendly. That would be great :D

Information for testing: The background of the login panel is not blurred in Internet Explorer. But it's not working in the current version either. I think it's not really that big a deal. As mentioned several times before a general UI IE patch (for a lot worse issues) will be inevitable at some point.

**Comment by gerhard-boden** [comment]
Style CI is the known error.. 

Anoher question: Do i need to push `Packages/Neos/TYPO3.Neos/Resources/Public/Styles/Login.css` or will that file be autogenerated?

**Comment by aertmann** [comment]
@gerhard-boden: things are easier to test if you push the `.css` files, just don't push the minified ones as they tend to conflict and aren't needed for testing

File: TYPO3.Neos/Resources/Public/Images/Login/Logo.svg [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+     width="200px" height="200px" viewBox="0 0 200 200" enable-background="new 0 0 200 200" xml:space="preserve">
+    <g>
+        <polygon fill="#26224C" points="132.984,37.5 112.342,52.662 112.342,84.378 132.984,113.791 	"/>
+        <polygon fill="#26224C" points="132.984,150.564 53.627,37.5 44.434,44.273 44.434,162.5 65.076,147.338 65.076,88.79
+                116.695,162.5 139.275,162.5 155.566,150.564 	"/>
+        <polygon fill="#00ADEE" points="65.076,88.79 65.076,147.338 44.434,162.5 67.016,162.5 87.658,147.338 87.658,121.038 	"/>
+        <polygon fill="#00ADEE" points="132.984,113.791 132.984,37.5 155.566,37.5 155.566,150.564 132.984,150.564 53.627,37.5
+                79.436,37.5 	"/>
+    </g>
+</svg>
```
Comment (aertmann) [inline TYPO3.Neos/Resources/Public/Images/Login/Logo.svg]: thanks!

Comment (aertmann) [inline TYPO3.Neos/Resources/Public/Images/Login/Logo.svg]: could be optimized..

    <svg xmlns="http://www.w3.org/2000/svg" width="200" height="200" viewBox="0 0 200 200"><path fill="#26224C" d="M132.984 37.5l-20.642 15.162v31.716l20.642 29.413M132.984 150.564L53.627 37.5l-9.193 6.773V162.5l20.642-15.162V88.79l51.619 73.71h22.58l16.291-11.936"/><path fill="#00ADEE" d="M65.076 88.79v58.548L44.434 162.5h22.582l20.642-15.162v-26.3M132.984 113.791V37.5h22.582v113.064h-22.582L53.627 37.5h25.809"/></svg>

**Comment by aertmann** [comment]
@gerhard-boden: would you mind providing some screen shots?

**Comment by gerhard-boden** [comment]
@aertmann thanks a lot for the review! Will make final adjustments and provide some screenshots for comparison to make it easier to see what this change actually does.

**Comment by gerhard-boden** [comment]
Can I safely delete `Packages/Neos/TYPO3.Neos/Resources/Public/Images/Login/ApplicationLogo.png`? It's the old logo.

I also took your  advice and optimized the new logo. It was 846 bytes and is now down to 419 bytes  ;) 

Here are two samples how the change improves the login screen behaviour @aertmann:

### First sample
(added image before ``.neos-login-box``, no CSS changes)

**Before:**
image

**Now:**
image


### Second sample
(added image within``.neos-login-box``,  no CSS changes)

**Before:**
image

**Now** (footer is displayed correctly below the fold)
image

### Third sample 
(more content added to the footer, centered footer content):

**Before:**
image

**Now:**
image

**Comment by aertmann** [comment]
@gerhard-boden thanks for the examples, although the topic is about making it small screen friendly, which you don't show.. the examples were however never meant to be supported, but there's no harm in doing so.. but it's cool if the box can have an arbitrary height now without problems

as for the `ApplicationLogo.png` feel free to remove it, just make sure it's not referenced anywhere

**Comment by aertmann** [comment]
and btw. 💯 for Mr. Meeseeks, just don't add more of him 😀

**Comment by gerhard-boden** [comment]
@aertmann The most obvious small screen optimization can be seen above when it comes to mobile screens https://github.com/neos/neos-development-collection/pull/567#issuecomment-225392620 

I also removed the old logo, searched the whole project, but no occurrences. 

But you're right the real purpose of this PR is something else: To make the login screen more adaptive to changes. I'll change the title of the PR to that. The bugfix label should remain, since it fixes the mobile view as well.

> and btw. :100: for Mr. Meeseeks, just don't add more of him :grinning:
Don't worry, after a Meeseeks has fulfilled his purpose he instantly disappears ;)
(in case someone is wondering what I'm babbling here, that thing is a character from a TV show that helps people with their tasks, like being a placeholder for actual content)

**Comment by gerhard-boden** [comment]
I just rebased on 2.2 to get rid of that StyleCI error

**Comment by knusperpixel** [comment]
@gerhard-boden here's the compressed background-image as talked about on #neos-core-dev
Filesize is down to 277kb (-57%)!

wallpaper22 compressed by tinypng

**Comment by kitsunet** [comment]
This needs a rebase but should be ready then. @gerhard-boden would you?

**Comment by gerhard-boden** [comment]
@kitsunet squashed 10 commits and put my work on top of the latest state of 2.2. Thanks for taking care of the merge!

**Comment by gerhard-boden** [comment]
For those who haven't heard about it yet:
This time symfony is messing with my tests:
https://github.com/symfony/symfony/issues/9815

So ignore the red `x` (even if it hurts)

**Pull request merged**